interim-jan-2021/slides-directories.md

```diff
@@ -0,0 +1,30 @@
+# Scenario 3: Directories of Encrypted Resolvers
+
+Three Parties:
+* Publisher: Curates a list of distinct resolvers
+* Client: Fetches the list from a trusted source
+* Resolvers: Identified in the list, ready for access by the client
+------------------------------------------------
+# Example use cases
+
+* A browser vendor wants to provide users with a list of resolvers to consider, curated by a trusted third party
+* An OS vendor wants to keep its list of trusted resolvers current without requiring a software update
+* A user wants to choose a resolver from a list offered by a network operator who they trust
+------------------------------------------------
+# Requirements
+
+* A list can be published by a trusted network
+* A list can be published at an HTTP URL
```

Comment:
From @jim-add

```suggestion
* A list can be published at an HTTP URL
* A signalling mechanism to describe resolver (protocol and policy) properties:
  eg DNSSEC or ECS support, DoH/DoT/DoQ/Do53, split DNS, filtering
```

Reply:
@jim-add I think these requirements should go in Section 2 (#34).

Reply:
Moved to https://github.com/ietf-wg-add/draft-add-requirements/pull/34#issuecomment-767111978 as agreed.